packages/nocodb/src/models/View.ts

```
@@ -256,328 +259,402 @@ export default class View implements ViewType {
+          `${CacheScope.VIEW}:${view_id}`,
+        );
+        return v;
+      });
+      if (copyFromView) {
+        Noco.appHooksService.emit(AppEvents.VIEW_DUPLICATE_COMPLETE, {
+          sourceView: copyFromView,
+          destView: view as ViewType,
+          req,
+          context,
+        });
+      }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)